addons/website_slides/controllers/mail.py

```diff
@@ -15,12 +16,18 @@ class SlidesPortalChatter(PortalChatter):
 
     @http.route(['/mail/chatter_post'], type='json', methods=['POST'], auth='public', website=True)
     def portal_chatter_post(self, res_model, res_id, message, **kw):
+        previous_post = request.env['mail.message'].search([('res_id', '=', res_id),
```

Comment:
can you add a small test ? (not a tour, just making manually the HTTP request on the endpoint)